### DISCUSSION TITLE
Pre-populated board

### DISCUSSION THREAD

**Discussion by dhaillant** [yaqwsx/PcbDraw#114] [Q&A] [answered]
Hello,
I'd like to know if there's a way to "pre-populate" a board?

For example, is there a way to show a board where all the SMD have already been soldered and the steps to be shown by populate would be only for the remaining THT components?

(ie. Not start with a blank PCB, but start with a PCB that has some components already)

Thank you.

**Comment by yaqwsx** [comment, marked as answer]
This was implemented in 9ca140d8188f7f9b36bcf0b2d99cdcb4ec029ada.
